packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/order/order.html

```diff
@@ -175,259 +175,286 @@ <h1 data-translate="telephony_alias_portability_order_title"></h1>
                 </div>
 
                 <div
-                    class="alert alert-warning"
-                    role="alert"
-                    data-ng-if="PortabilityOrderCtrl.isSpecialNumber && !PortabilityOrderCtrl.isSDA"
-                    data-translate="telephony_alias_portability_order_sda_warning"
-                ></div>
-
-                <div
-                    class="form-group"
-                    data-ng-if="PortabilityOrderCtrl.isSpecialNumber"
+                    data-ng-if="PortabilityOrderCtrl.isSpecialNumber && !PortabilityOrderCtrl.canOrderSpecialPortability"
```

Comment:
Seems GitHub fails to show the correct diff.
I've just added a `div` if `svaWallet` isn't valid and add a `div` wrapper for the existing form part (if sva valid)...